src/components/navigation/Navbar.test.tsx

```diff
@@ -9,15 +9,15 @@ import { MemoryRouter, Link } from "react-router-dom";
 describe("Navbar", () => {
   it("should render", async () => {
     renderWithProviders(<Navbar />);
-    expect(await screen.findByRole("banner")).toBeInTheDocument();
+    expect(await screen.findByTestId("navbar")).toBeInTheDocument();
```

Comment:
We should be avoiding test-ids in components people will be reusing.

See comment on Navbar.tsx file.